package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.storage",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Oh sorry, you need to remove this two. It will come as part of releasing, and it needs to be a feature bump not a fix bump.